### PR TITLE
Added search to `edit-view`.

### PR DESCRIPTION
**Pull request by ghost**
Search now is added to edit views of objects (e.g: supports).

connected to: #1610

### CODE REVIEW

File: src/ralph/admin/templates/admin/search_form.html [hunk 1]
```diff
@@ -1,44 +1,30 @@
...
       </div>
-      <div class="small-3 columns">
+      <div class="small-2 columns">
         <button type="submit" class="button expand search">
           <i class="fa fa-search"></i> {% trans 'Search' %}
         </button>
       </div>
     </div>
   </form>
-{#   <div id="toolbar"> #}
```
Comment (mkurek) [inline src/ralph/admin/templates/admin/search_form.html:19]: if it's not necessary remove it

**Comment by vi4m** [comment]
Is this really so important to implement this feature right now?  We still want to have left-sidebar search like in old version.

**Comment by ghost** [comment]
@vi4m removed context-switcher, only added search to edit views.

**Comment by mkurek** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**